lib/__init__.py

```diff
@@ -0,0 +1,19 @@
+__version__ = '1.1.0'
```

Comment:
It may work without it but adding the __init__.py files tells python what the structure of the project is - what can be imported from where. If you don't import cross folders it probably works without. I mainly added it after reading up on it and consider it cleaner to add them everywhere or remove them altogether but please ignore me if anyone here knows more about __init__.py...